docs/tutorials/creating-a-runtime-module.md

```diff
@@ -2,46 +2,68 @@
+If you haven't already, follow the
+[fast installation](getting-started/installing-substrate#fast-installation) for
+getting started on Substrate. It will provide you with all the development
+resources you need to build Substrate and the
+[`substrate-up` scripts](getting-started/using-the-substrate-scripts) for
+quickly setting up new nodes.
 
-## Create your Node
+### Cloning Node and Module Template to Start
 
-We'll begin by creating a brand new node template. If you've followed other runtime tutorials, you've probably started this way before:
-
-```bash
-substrate-node-new modular-chain <your name>
-```
-
-In this tutorial, we're not going to write our module directly as part of the node template, but rather as a separate Rust crate. This approach provides a few advantages:
+We're not going to write our module directly as part of the node template, but
+rather as a separate Rust crate. This approach provides a few advantages:
 
 1. Our module can be easily imported into other nodes in the future.
-2. Our module can be published to places like GitHub or Crates.io without publishing our entire node in the same place.
+2. Our module can be published to places like GitHub or Crates.io without
+publishing our entire node in the same place.
 
-However, we do still need to set up a Substrate node to actually run our module.
+However, to effectively develop and test with your own runtime module, you need
+to first setup a single-node Substrate blockchain. Then inside the node runtime
+add in your own additional module.
 
-## Clone the Module Template Repository
+We'll begin from cloning the necessary repositories:
 
-So if our module isn't going to live in the node template, where _is_ it going to live?
+```bash
+cd my-playground-folder
+# clone from a standard node template
+git clone https://github.com/substrate-developer-hub/substrate-node-template.git my-node
+# clone from a standard module template
+git clone https://github.com/substrate-developer-hub/substrate-module-template.git my-module
+# kick off the node compilation
+cd my-node
+cargo build --release
+```
 
-If you aren't familiar with creating Rust libraries or using them in other Rust projects, you can get a more in-depth explanation by reading [the Cargo book](https://doc.rust-lang.org/cargo/guide/creating-a-new-project.html).
+The compilation of the node may take up to 30 minutes, depending on your hardware.
+So let us kickstarted it first.
 
-While it's perfectly possible to create a new Substrate runtime module from scratch, the fastest way to get started is to clone a Substrate module template we have created for you:
+Now, if our module isn't going to live in the node template, where _is_ it going
+to live? If you aren't familiar with creating Rust libraries or using them in
```

Comment:
It would be good to clearly and directly answer the question you pose here, and then tell them to learn more from a link.

Otherwise, to me right now, it reads a bit like an open ended question.